### ISSUE TITLE
Question: Is it possible to pickup folder as a path?

### ISSUE THREAD

**Issue by isabsent** [dibakarece/AndroidFileExplorer#4]
I need to select and pickup the selected folder path. Is it possible with your library?

**Comment by dibakarece** [comment]
Yes, you can do it.. but for this you need to customize  this library and need to define your specific folder. On item click event need check wither this is the specific folder that you wants to then return that folder else proceed  with click.
In 'DirectoryFragment.java' just check the below code 

            listView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
                @Override
                public void onItemClick(AdapterView<?> adapterView, View view,
                                        int i, long l) {
